package.json

```diff
@@ -23,6 +23,7 @@
     "builder": "~2.1.4",
     "builder-victory-component": "~0.0.5",
     "d3-scale": "^0.3.0",
+    "lodash": "^3.10.1",
```

Comment:
This isn't necessary as it is already included via the `builder-victory-component` archetype. Please remove.